tools/downloader/topologies.py

```diff
@@ -0,0 +1,82 @@
+import os
+import argparse
+import sys
+
+from pathlib import Path
+from argparse import Namespace
+
+import common
+import downloader
+import converter
+
+_download_dir = Path(os.getenv('OPEN_MODEL_ZOO_MODELS_DIR', Path('.').resolve() / 'models'))
+_cache_dir = Path(os.getenv('OPEN_MODEL_ZOO_CACHE_DIR', _download_dir / 'models_cache'))
+
+def load_model(model_name, precision='FP32'):
+    '''
+    Downloads target model to the folder specified in _download_dir variable.
+    If there is already downloaded model - retrieves the model from cache.
+
+    Parameters
+    ----------
+        model_name
+            Target model name.
+        precision
+            Target model precision. By default, FP32 precision is used.
+
+    Returns
+    -------    
+    An object of the Topology class which stores information about the downloaded model.
+    '''
+    parser = argparse.ArgumentParser()
+    args = Namespace(all=False, list=None, name=model_name, print_all=False)   
+    topology = common.load_models_from_args(parser, args)[0]
+
+    model_dir = _download_dir / topology.subdirectory
+
+    downloader.main(['--name=' + model_name, '--precisions=' + precision,
```

Comment:
Please check if we can download models without FP32 precision. Maybe it make sense to make default value of precision as None